Smt/Constants.lean

```diff
@@ -27,53 +23,4 @@ def defNatSub (s : Solver) : Solver :=
    defineFun s "Nat.sub" [("x", `"Nat"), ("y", `"Nat")] (`"Nat") $
     `"ite" • (`"<" • `"x" • `"y") • ``"0" • (`"-" • `"x" • `"y")
 
-/-- Map from known constants to corresponding SMT constant/function symbols. -/
-def knownConsts : Std.HashMap Expr (Option Expr) :=
```

Comment:
I'm glad to see this massive table gone :)

Reply:
I would also like to get rid of the other one in `Util.lean`, but I don't have enough cycles to think of a clean way to utilize attributes in `Query.lean` 😔.